.gitignore

```diff
@@ -33,3 +33,6 @@ docs/_build
 /.pytest_cache/
 /.eggs
 /.vscode
+.venv
```

Comment:
Perhaps `/.venv` to only ignore these on the top level?

Reply:
I will sometimes use a venv in the docs/ folder so I can separate docs dependencies.